README.md

```diff
@@ -0,0 +1,27 @@
+# bioutils
+
+A repository for generally useful bioinformatics tools
+
+# Description
+
+Here, we will consolidate scripts for commonly-used functions such as converting filetypes or calculating descriptive statitics.
+
+# Usage
+
+1. Go to https://github.com/maragkakislab/bioutils and fork the repository to your own Github account
+2. Go to a directory of choice to install bioutils, suggest `~/dev`: `cd ~/dev`
```

Comment:
2. Clone your forked repository to a directory of your choice and then change into the cloned repository e.g. `cd ~/dev/bioutils`